src/amo/components/RatingManager/styles.scss

```diff
@@ -60,3 +60,16 @@
   margin-top: 12px;
   width: 100%;
 }
+
+.RatingManager-savedRating {
```

Comment:
Could we pull this logic into a new component like `RatingManagerNotice` so that it can be shared?